### ISSUE TITLE
Twilio's Authy

### ISSUE THREAD

**Issue by angelineflorajohn** [raycast/extensions#10544]
### Extension

https://www.raycast.com/guga4ka/authy

### Description

I am constantly getting the "corrupted" error when I try to fetch the OTP tokens as shown in the screenshot.
![image](https://github.com/raycast/extensions/assets/25217498/9a7a2fcd-8636-42cb-8db5-ff41d2220f6e)

I have tried to quit and restart Raycast, also tried to uninstall and install extension.
Kindly advice. 

### Steps To Reproduce

1. Installed Twilio's Authy extension from Raycast extension store.
2. Entered Twilio account details.
3. Approved the Request on other device.
4. Refreshed the request status on Raycast.
5. Shows the tokens as corrupted.
![image](https://github.com/raycast/extensions/assets/25217498/5ff5d1b2-be8f-4c38-8745-adacd62a5b4a)
![SCR-20240203-ikgt](https://github.com/raycast/extensions/assets/25217498/16dbb9e1-978e-445b-9097-46a315889242)

### Current Behaviour

![SCR-20240203-ikgt](https://github.com/raycast/extensions/assets/25217498/f19df8aa-7dd9-4206-be15-028d435a9eb9)

### Expected Behaviour

![SCR-20240203-ilek](https://github.com/raycast/extensions/assets/25217498/ef8faa3e-2859-4cbc-a0c9-2d3be9f7d945)

**Comment by guga4ka** [comment]
Looks like your backup password is wrong. Did extension show you popup about it?

**Comment by angelineflorajohn** [comment]
Yes, the extension did ask me for my AuthyID and Authy backup password. Once I entered the password, I got the "Approval request has been sent screen" on Raycast as shown below.
![SCR-20240204-kavn](https://github.com/raycast/extensions/assets/25217498/c7df39d9-e8b2-4691-9459-8295892b8fb3)

Once I approve the request on Authy, I am asked to enter "YES" to add the new device.
![SCR-20240204-kazr](https://github.com/raycast/extensions/assets/25217498/c41c2a6a-66dd-4b56-98f5-30203d2396b1)

Once the device has been added, I come back to Raycast and refresh the status. As shown below, you can see that most of the tokens show up as "corrupted" but there were two apps which showed valid tokens.
![SCR-20240204-kbiy](https://github.com/raycast/extensions/assets/25217498/4232af08-ab7c-4e3f-8eb2-715b4471bdc3)

I hope the above mentioned steps are helpful in debugging the issue.

**Comment by angelineflorajohn** [comment]
@guga4ka is there any update on this?

**Comment by fo-fo** [comment]
I looked into this (at the code and API level), and was able to fix it by changing the backup key in Authy.

Based on what I saw, It seems highly likely that that this is actually a bug in Authy and it's possible to get into a situation where some of the seeds stored in the server are not encrypted with the current backup key.

Out of curiosity, it would be interesting for someone with the issue to try to sync the tokens to a new device with Authy. I can't test it anymore since I already changed my backup key and no longer have this issue.
